extensions/wikia/RTE/RTE.class.php

```diff
@@ -682,7 +687,7 @@ static function onEditingPreferencesBefore($user, &$preferences) {
 	/**
 	 * Modify values returned by User::getGlobalPreference() when wysiwyg is enabled
 	 */
-	static public function userGetOption($options, $name, &$value) {
+	static public function userGetOption($options, $name, &$value): bool {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)